tar/tarsnap.1-man.in

```diff
@@ -1190,3 +1190,7 @@ is run with standard input, standard output, and standard error
 redirected and inside a chroot where terminal devices are not
 exposed, ^Q will not be mapped to SIGQUIT and will consequently not
 trigger the truncation of the current archive.
+.PP
```

Comment:
I wouldn't say that
> To archive a file called @foo, @@foo, or -foo you must specify it as ./@foo, ./@@foo, or ./-foo, respectively.

or 
> There are alternative long options for many of the short options that are deliberately not documented.

were bugs either.  But ok, I'll put it somewhere else.